app/src/main/java/org/oppia/app/walkthrough/topiclist/WalkthroughTopicViewModel.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.app.walkthrough.topiclist
+
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import javax.inject.Inject
+
+/** The ViewModel for [WalkthroughTopicListFragment]. */
+class WalkthroughTopicViewModel @Inject constructor(
```

Comment:
Implementation in this file is not correct. You can optimise this. For reference you can check:
`CompletedStoryListViewModel`